detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ForbiddenComment.kt

```diff
@@ -38,7 +38,13 @@ class ForbiddenComment(config: Config = Config.empty) : Rule(config) {
     )
 
     @Configuration("forbidden comment strings")
-    private val values: List<String> by config(listOf("FIXME:", "STOPSHIP:", "TODO:"))
+    @Deprecated("Use `valuePatterns` instead")
```

Comment:
Somehow this should be mentioned I feel:
```suggestion
    @Deprecated("Use `valuePatterns` instead, make sure you escape your text for Regular Expressions.")
```
otherwise people might just change `values` to `valuePatterns` and expect it to "just work".